tools/commands/build_example.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import re
+import subprocess
+import sys
+import os
+import commands.command_utils as command_utils
+
+
+def parse_args(args):
+    parser = command_utils.get_options_parser(plugins=True, exclude=True, run_on_changed_packages=True, base_sha=True)
+    return parser.parse_args(args)
+
+
+def _build_examples(plugin):
+    print(f'============= Building for {plugin} =============')
```

Comment:
For readability, `{plugin}` should display the basename only.

Reply:
I fixed it as your opinion